core/rpc/client/user/http/convert.go

```diff
@@ -14,6 +14,10 @@ import (
 
 // convertTx converts a transaction to a httpTx.TxTransaction
 func convertTx(tx *transactions.Transaction) *httpTx.TxTransaction {
+	if tx == nil {
+		return nil
+	}
```

Comment:
I have no problem with this, for safety, but is there somewhere this is expected presently?  Can leave regardlesss, but I don't see it being used to handle the nil unconfirmed thing back from server.